packages/from-query/src/generate.ts

```diff
@@ -129,17 +149,32 @@ export default (OPTIONS: IFromQueryOptions): (ir: IOperation) => string => {
                 interfaceName: name
               }
             }
+
             return name;
           });
-          return printField(
-            this.generateSelectionName(selection),
-            selection.typeDefinition,
-            selection
-          );
+
+          const selectionHasDirectives = hasDirectives(selection.directives);
+          const fragmentsHaveDirectives = !!selection.fragments.some(frag => hasDirectives(frag.directives))
```

Comment:
```suggestion
          const fragmentsHaveDirectives = selection.fragments.some(frag => hasDirectives(frag.directives))
```

`!!` is unnecessary here as `Array.prototype.some` will always return a `bool`